easybuild/easyblocks/w/wps.py

```diff
@@ -93,8 +95,14 @@ def configure_step(self):
         # WRF dependency check
         wrf = get_software_root('WRF')
         if wrf:
-            majver = get_software_version('WRF').split('.')[0]
-            self.wrfdir = os.path.join(wrf, "WRFV%s" % majver)
+            wrfver = get_software_version('WRF')
+            majver = wrfver.split('.')[0]
+            if majver == '3':
+                self.wrfdir = os.path.join(wrf, "WRFV%s" % majver)
+            elif majver == '4':
```

Comment:
@andreas-h Please use LooseVersion here rather than doing string comparison on the major version

Reply:
addressed by https://github.com/andreas-h/easybuild-easyblocks/pull/4